queue/python/queue.py

```diff
@@ -0,0 +1,46 @@
+# Implementation of queue using Linked List:
+
+class Node:
+    def __init__(self, data):
+       self.data = data
+       self.next = None
+ 
+class Queue:
+    def __init__(self):
+        self.head = None
+        self.last = None
+ 
+    def enqueue(self, data):
+        if self.last is None:
+            self.head = Node(data)
+            self.last = self.head
+        else:
+            self.last.next = Node(data)
+            self.last = self.last.next
+ 
+    def dequeue(self):
+        if self.head is None:
+            return None
+        else:
+            to_return = self.head.data
+            self.head = self.head.next
+            return to_return
+ 
+a_queue = Queue()
+while True:
+    print('enqueue <value>')
+    print('dequeue')
+    print('quit')
+    do = input('What would you like to do? ').split()
+ 
+    operation = do[0].strip().lower()
+    if operation == 'enqueue':
+        a_queue.enqueue(int(do[1]))
+    elif operation == 'dequeue':
+        dequeued = a_queue.dequeue()
+        if dequeued is None:
+            print('Queue is empty.')
+        else:
+            print('Dequeued element: ', int(dequeued))
+    elif operation == 'quit':
+        break
```

Comment:
Line contains following spacing inconsistencies:
- No newline at EOF.

*Origin: SpaceConsistencyBear, Section: `all.pyjava`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmp_mrnlsdu/queue/python/queue.py
+++ b/tmp/tmp_mrnlsdu/queue/python/queue.py
@@ -43,4 +43,4 @@
         else:
             print('Dequeued element: ', int(dequeued))
     elif operation == 'quit':
-        break+        break
```